wallets/provider-walletconnect-2/package.json

```diff
@@ -27,18 +27,18 @@
     "@rango-dev/signer-evm": "^0.21.1-next.0",
     "@rango-dev/signer-solana": "^0.21.1-next.0",
     "@rango-dev/wallets-shared": "^0.21.1-next.3",
-    "@solana/web3.js": "^1.67.2",
-    "@walletconnect/encoding": "^1.0.2",
-    "@walletconnect/modal": "^2.6.2",
-    "@walletconnect/sign-client": "^2.10.4",
-    "@walletconnect/utils": "^2.10.4",
+    "@solana/web3.js": "1.67.2",
```

Comment:
Do we need this change?

Reply:
Let me ask in this way, why we need this changes on WC2 provider while the PR is about Safe? Do we need these changes for doing the work?

Reply:
Please check my second bullet in PR description > summary.